features/user/managing_customers/browsing_customers.feature

```diff
@@ -0,0 +1,17 @@
+@managing_customers
+Feature: Browsing customers
+    In order to see all customers in the store
+    As an Administrator
+    I want to browse customers
+
+    Background:
+        Given the store has customer "f.baggins@example.com"
+        And the store has customer "mr.banana@example.com"
```

Comment:
Why not use ``t.lannister@example.com`` or ``geralt.of.rivia@example.com`` to make it the coolest feature in the whole *Sylius*? :smile: